test/athens/bfs_test.cljc

```diff
@@ -31,12 +35,15 @@
   (let [db common-db/empty-db]
     (is (= [#:op{:type :page/new, :atomic? true, :args {:page/title "Welcome"}}
             #:op{:type :block/new, :atomic? true, :args {:block/uid "block-1", :block/position {:page/title "Welcome", :relation :last}}}
-            #:op{:type :page/new, :atomic? true, :args {:page/title "Welcome"}}
             #:op{:type :block/open :atomic? true :args {:block/uid "block-1" :block/open? false}}
-            #:op{:type :block/save, :atomic? true, :args {:block/uid "block-1", :block/string "block with link to [[Welcome]]"}}]
+            #:op{:type :page/new, :atomic? true, :args {:page/title "property key"}}
+            #:op{:type :block/new, :atomic? true, :args {:block/uid "prop-1", :block/position {:page/title "Welcome", :relation {:page/title "property key"}}}}
+            #:op{:type :block/save, :atomic? true, :args {:block/uid "block-1", :block/string "block with link to [[Welcome]]"}}
+            #:op{:type :block/save, :atomic? true, :args {:block/uid "prop-1", :block/string "property value"}}]
            (bfs/internal-representation->atomic-ops db tree-with-page nil)))
 
-    (is (= [#:op{:type :block/new, :atomic? true, :args {:block/uid "eaa4c9435", :block/position {:page/title "title", :relation :first}}}
+    (is (= [#:op{:type :page/new, :atomic? true, :args #:page{:title "title"}}
```

Comment:
IR now creates the location because of the changes in `src/cljc/athens/common_events/graph/ops.cljc`. I think this is correct.